xcodeproj/internal/xcodeproj_macro.bzl

```diff
@@ -19,6 +19,7 @@ def _normalize_build_setting(flag):
 def xcodeproj(
         *,
         name,
+        adjust_dependencies_for_xcode_previews = True,
```

Comment:
Should this even be configurable? For schemes it made sense, I'm not sure it makes sense here.

Reply:
Would it be possible to set a global default without it? fwiw we haven't needed to set this to `False` any longer need to use this as we fixed our top-level target issues

Reply:
I'm proposing that there would be no need to disable this. Unlike the scheme way, I can't think of a reason, outside of potential small performance gains, to disable it.